internal/armada/server.go

```diff
@@ -129,3 +137,10 @@ func Serve(config *configuration.ArmadaConfig, healthChecks *health.MultiChecker
 func createRedisClient(config *redis.UniversalOptions) redis.UniversalClient {
 	return redis.NewUniversalClient(config)
 }
+
+func getCancelJobsBatchSize(config *configuration.ArmadaConfig) int {
+	if config.CancelJobsBatchSize > 0 {
```

Comment:
Is it not worth putting in an upper limit? Would there not be a problem if we asked the server to cancel jobs in batches of 1000000 or so?